body.h

```diff
@@ -101,5 +101,6 @@ struct Body
 struct Body *mutt_new_body(void);
 int mutt_copy_body(FILE *fp, struct Body **tgt, struct Body *src);
 void mutt_free_body(struct Body **p);
+int strict_cmp_bodies(const struct Body *b1, const struct Body *b2);
```

Comment:
mutt_body_strict_cmp? mutt_strict_cmp_body? But please avoid plural, it makes it even harder to spot similar function

Reply:
> But please avoid plural

I hadn't thought about that.  Noted.